CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ We do our work on a feature branch based on master. When you open a pull request
 
 This is also a good opportunity to verify that all the appropriate image versions were bumped. It is important to keep in mind which images extend from where ([Image Dependecies](https://github.com/DataBiosphere/terra-docker#image-dependencies)). Make sure to bump the version of the image you are changing and update its reference in all images that extend from it.
 
-You can also run `amm ./updateVersions.sc <image name> <release note>` to upgrade derived images.
+You can also run `amm ./updateVersions.sc <image name> <release note>` to upgrade derived images. If you're updating `terra-jupyter-aou` image, please make a pull request similar to [this PR](https://github.com/DataBiosphere/leonardo/pull/1612) (less updating `terra_jupyter_aou_old`).
```

Comment:
```suggestion
You can also run `amm ./updateVersions.sc <image name> <release note>` to upgrade derived images. If you're updating `terra-jupyter-aou` image, please make a pull request similar to [this PR](https://github.com/DataBiosphere/leonardo/pull/1612) (unless updating `terra_jupyter_aou_old`).
```
assuming it's a typo 😊

Reply:
oh, I meant the change for `updating `terra_jupyter_aou_old`` is irrelevant for this purpose